library/general/src/lib/installation/autoinst_profile/section_with_attributes.rb

```diff
@@ -123,8 +123,11 @@ class SectionWithAttributes
       class << self
         # Description of the attributes in the section.
         #
-        # To be defined by each subclass. Each entry contains a hash with the
-        # mandatory key :name and an optional key :xml_name.
+        # To be defined by each subclass. Each entry contains a hash with:
+        #   * :name        : the mandatory key
+        #   * :xml_name    : an optional alternative key
```

Comment:
np: Actually, it is not exactly an "alternative", but the name of the element in the XML profile.